pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup

```diff
@@ -9,7 +9,7 @@ hv=`cat /run/eve-hv-type`
 default_cgroup_cpus_limit=1
 
 default_cgroup_memory_limit=838860800 #800M
-EVESERVICES="sshd edgeview wwan wlan lisp guacd pillar vtpm watchdog xen-tools newlogd memlogd"
+EVESERVICES="sshd eve-edgeview wwan wlan lisp guacd pillar vtpm watchdog xen-tools newlogd memlogd"
```

Comment:
Please don't forget to reflect changed directory output in your own documentation: ./pkg/memory-monitor/README.md:18

Reply:
Do you mean the example of the cgroups hierarchy?

Reply:
Done